common-content/en/module/html-css/what-is-css/index.md

```diff
@@ -34,18 +34,20 @@ selector {
 }
 ```
 
-Selectors look like this: `p { background: red }`. The selector is the `p` in this case. This `p` selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a ruleset.
+Rules look like this: `p { background: red; }`. The selector is the `p` in this case. This `p` selects all elements of the DOM tree called `p` and sets a rule about how to show them. Selectors always come first in a rule.
 
 A property is a quality, a characteristic. A value is the amount.
 
 For a person, you might say their 'age' property has the value of 40. For an element, you might set the `width` property to the value of `100%`.
 
+Within a rule, multiple properties may be set to values, and each property-value pair must be followed by a semi-colon (`;`), e.g. `p { background: red; color: blue; }`.
```

Comment:
No it's a ruleset. 
```suggestion
Each ruleset may have many declarations, or rules. A declaration is a property paired with a value. Each property-value pair must be followed by a semi-colon (`;`), e.g. `p { background: red; color: blue; }`.
```

Reply:
Interesting, do you have a source for this?

AFAICT the world treats "rule" and "ruleset" interchangeably, so I was just picking one to be consistent rather than open up the question of why there are two terms... I don't mind which we pick, but I'm not sure the "declaration == rule" holds.

The spec claims "rule set" and "rule" are interchangeable:
> A rule set (also called "rule")

(https://www.w3.org/TR/CSS2/syndata.html#rule-sets), and that the things inside the `{}`s are "declarations" (not "rules") (https://www.w3.org/TR/CSS2/syndata.html#declaration).

MDN agrees:
> A [selector list](https://developer.mozilla.org/en-US/docs/Web/CSS/Selector_list) and an associated declarations block, together, are called a ruleset, or often a rule.
>
> A CSS ruleset (or rule) is visualized in the diagram below.

(https://developer.mozilla.org/en-US/docs/Web/CSS/Syntax#css_rulesets)

Reply:
Am I out of date? Crivens. Overruled! 😉

Reply:
Updated in [d15b267](https://github.com/CodeYourFuture/curriculum/pull/943/commits/d15b26759f625be577e9f5976d0908abe16b5127) to meet somewhere in the middle :)